estimation/launch/mj_estimation.launch

```diff
@@ -0,0 +1,47 @@
+<launch>
+
+	<arg name="simulation" value="true"/>
+
+	<arg name="mujoco_cam_file" value="$(find estimation)/config/mujoco_cam.yaml"/>
+	<arg name="logitech_cam_file" value="$(find estimation)/config/cTw_cam0.yaml"/>
+
+	<arg name="cam_file" if="$(arg simulation)" value="$(arg mujoco_cam_file)" />
+  <arg name="cam_file" unless="$(arg simulation)" value="$(arg logitech_cam_file)" />
+  
+  <group ns="mushr">
+    <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>
```

Comment:
If this file is included from another launch file, this line will throw an error.